code/pizzabot.js

```diff
@@ -1,7 +1,47 @@
+
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
+const margherita = "Margherita Pizza"
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+
+const calculateTotalCost = (orderQuantity) => {
+    orderTotal = (orderQuantity * pizzaPrice)
```

Comment:
No need for extra pair of parentheses here, `orderTotal = orderQuantity * pizzaPrice;` is OK :)